test/fuzz/luaL_loadbuffer/serializer.cc

```diff
@@ -5,6 +5,161 @@
  */
 #include "serializer.h"
 
+#include <stack>
+#include <string>
+
+namespace {
+
+/**
+ * Class that controls id creation for counters. Basically, a global variable
+ * that guarantees to be incremented.
+ */
+class CounterIdProvider_ {
+public:
+	/** Returns number of id provided. */
+	std::size_t count()
+	{
+		return id_;
+	}
+
+	/** Returns a new id that was not used after last clean(). */
+	std::size_t next()
+	{
+		return id_++;
+	}
+
+	/** Cleans history. Should be used to make fuzzer starts independent. */
+	void clean()
+	{
+		id_ = 0;
+	}
+
+private:
+	std::size_t id_ = 0;
+};
+
+/** A singleton for cycles and not anonymous function definitions. */
+CounterIdProvider_&
+common_id_provider()
+{
+	static CounterIdProvider_ common;
+	return common;
+}
+
+/**
+ * A singleton for anonymous function definitions. It must be a separate
+ * object to make possible count only anonymous functions.
+ */
+CounterIdProvider_&
+anonfunc_id_provider()
+{
+	static CounterIdProvider_ anonfunc;
+	return anonfunc;
+}
+
+std::string
+get_counter_name(const std::string &prefix, std::size_t id)
+{
+	return prefix + std::to_string(id);
+}
+
+std::string
+get_counter_initialization(const std::string &counter_name)
+{
+	std::string retval = "local ";
+	retval += counter_name;
+	retval += " = 0\n";
+	return retval;
+}
+
+std::string
+get_counter_increment(const std::string &counter_name)
+{
+	std::string retval = counter_name;
+	retval += " = ";
+	retval += counter_name;
+	retval += " + 1\n";
+	return retval;
+}
+
+std::string
+get_condition(const std::string &counter_name, const std::string &then)
+{
+	std::string retval = "if ";
+	retval += counter_name;
+	retval += " > ";
+	retval += std::to_string(kMaxCounterValue);
+	retval += " then ";
+	retval += then;
+	retval += " end\n";
+	return retval;
+}
+
+/**
+ * Block may be placed not only in a cycle, so specially for cycles there is
+ * a function that will add break condition and counter increment.
+ */
+std::string
+BlockToStringCycleProtected(const Block &block, const std::string &counter_name)
+{
+	std::string retval = get_condition(counter_name, "break");
+	retval += get_counter_increment(counter_name);
+	retval += ChunkToString(block.chunk());
+	return retval;
+}
+
+/**
+ * DoBlock may be placed not only in a cycle, so specially for cycles there is
+ * a function that will call BlockToStringCycleProtected().
+ */
+std::string
+DoBlockToStringCycleProtected(const DoBlock &block,
+			      const std::string &counter_name)
+{
+	std::string retval = "do\n";
+	retval += BlockToStringCycleProtected(block.block(), counter_name);
+	retval += "end\n";
+	return retval;
+}
+
+std::string
+FuncBodyToStringReqProtected(const FuncBody &body,
+			     const std::string &counter_name)
+{
+	std::string body_str = "( ";
+	if (body.has_parlist())
+		body_str += ParListToString(body.parlist());
+	body_str += " )\n\t";
```

Comment:
I think whitespaces after `(` and before `)` could be omitted.

Reply:
Actually I took these lines from `FuncBodyToString`: https://github.com/dmitrylala/tarantool/blob/b2818459e2cc5f2da4bc0341057664947b9b8e1b/test/fuzz/luaL_loadbuffer/serializer.cc#L358-L361

If you still want me to change it, I will

Reply:
I agree with Sergey that these whitespaces are excess.